src/pages/home/report/ReportActionCompose.js

```diff
@@ -116,6 +119,14 @@ const defaultProps = {
     ...withCurrentUserPersonalDetailsDefaultProps,
 };
 
+/**
+  * Return amount of emoji. For small screen we can fit 3 items and for large we show up to 5 items
+ * @param {Number} numRows
+ * @param {Boolean} isEmojiPickerLarge
+ * @returns {Number}
+ */
+const countAmountOfEmoji = (numRows, isEmojiPickerLarge) => (isEmojiPickerLarge ? Math.max(numRows, 5) : Math.max(numRows, 3));
```

Comment:
What do you think of this?
```suggestion
const getEmojiRowCount = (numRows, isEmojiPickerLarge) => (isEmojiPickerLarge ? Math.max(numRows, 5) : Math.max(numRows, 3));
```